pandas/core/strings.py

```diff
@@ -678,17 +678,45 @@ def str_replace(arr, pat, repl, n=-1, case=None, flags=0, regex=True):
 
 def str_repeat(arr, repeats):
     """
-    Duplicate each string in the Series/Index by indicated number
-    of times.
+    Duplicate each string repeated by indicated number of times.
+
+    Duplicate each string in the Series/Index by indicated number of times.
+    A passed value of zero or negative integer will return an empty string.
 
     Parameters
     ----------
-    repeats : int or array
-        Same value for all (int) or different value per (array)
+    repeats : int or array-like
+        Same value for all (int) or different value per (array).
 
     Returns
     -------
-    repeated : Series/Index of objects
+    Series or Index
+        Series or Index of repeated string objects specified by
+        input parameter repeats.
+
+    Examples
+    --------
+    >>> s = pd.Series(['a', 'b', 'c', 'd', 'e'])
```

Comment:
I think having 5 cases that illustrate exactly the same is not very useful. Can we have:
- A single letter (may be two or three, for the next example to show different number of repetitions)
- A word
- A `NaN`
- A number

Can you also show the value of `s`, so it's easier to compare with the result of `str.repeat`